spec/markdown-preview-spec.coffee

```diff
@@ -349,6 +349,25 @@ describe "Markdown preview package", ->
 
       runs -> expect(preview.find('atom-text-editor')).toExist()
 
+  describe "when there is an image with a relative path and no directory", ->
+    it "does not alter the image src", ->
+      runs ->
+        atom.project.removePath(projectPath) for projectPath in atom.project.getPaths()
```

Comment:
This does not need to be in a `runs` block since it is right at the beginning of an `it`, it can just be right under the `it`